site/data/refDataApi.ts

```diff
@@ -77,7 +77,7 @@ export const fetchServices = async (input: FetchServicesInput) => {
         method: "GET",
     });
 
-    const parseResult = z.array(serviceSchema).safeParse(await res.json());
+    const parseResult = z.array(serviceApiResponseSchema).safeParse(await res.json());
 
     if (!parseResult.success) {
```

Comment:
After testing found a duplicate 
<img width="720" alt="Screenshot 2023-08-08 at 15 47 09" src="https://github.com/Department-for-Transport-Disruptions/create-disruptions-data/assets/59929701/f4aeb4a5-1996-42e2-9170-e5bb5b08bec4">

Reply:
<img width="543" alt="Screenshot 2023-08-08 at 15 53 25" src="https://github.com/Department-for-Transport-Disruptions/create-disruptions-data/assets/59929701/f74a8bb9-2498-4ec5-ba63-e020444fdf14">

Reply:
Steven is happy with output as discussed